ui/pages/shield-plan/shield-plan.tsx

```diff
@@ -343,7 +350,16 @@ const ShieldPlan = () => {
     return details;
   }, [t, isTrialed, selectedProductPrice]);
 
+  const planDetailsRewardsText = useMemo(() => {
+    const points =
+      selectedPlan === RECURRING_INTERVALS.year ? '10,000' : '1,000';
```

Comment:
shouldn't this come from subscription pricing api? @tanguyenvn

Reply:
uses rewardsShield hook now https://github.com/MetaMask/metamask-extension/pull/38379/files#diff-9d5f8d40f5ff16a52171342e5c42a4f3672a168cee24f16d217dac6d050c5d47R107